src/Admin/Settings/API.php

```diff
@@ -96,38 +96,37 @@ public function render_text_field( array $field ) {
 	 * @since  1.3.0
 	 * @access public
 	 *
-	 * @return mixed
+	 * @return string
 	 */
 	public function render_group_field( array $group ) {
 		$settings    = Helpers::get_settings();
-		$toggle      = ! ! $group['toggle'];
+		$toggle      = $group['toggle'];
 		$fields      = $group['fields'];
 		$field_value = ! empty( $settings[ $group['slug'] ] ) ? $settings[ $group['slug'] ] : false;
 		$is_checked  = checked( $field_value, true, false );
-		$domain_name = ! empty( $settings['domain_name'] ) ? $settings['domain_name'] : '';
 		ob_start();
 		?>
 		<div class="plausible-analytics-admin-field">
 			<div class="plausible-analytics-admin-field-header">
 				<label for="">
-					<?php esc_html_e( $group['label'], 'plausible-analytics' ); ?>
+					<?php echo $group['label']; ?>
```

Comment:
These are translated elsewhere, you can't pass variables into translation functions in WP.